server2/handlers/like.handler.ts

```diff
@@ -0,0 +1,77 @@
+import {Request, Response} from "express"
+import {IMainController}   from "../controllers/main.controller";
+import * as dayjs          from "dayjs";
+import {IUser}             from "../../shared/types/Entities/IUser";
+
+
+// Get all likes
+export const getLikesHandler = async function (this: IMainController, req: Request, res: Response) {
+	try {
+		const likes = await this.likeController.getLikes()
+		return res.send(likes);
+
+	} catch (e) {
+		return res.status(404).send({msg: 'like has not found' + e})
+	}
```

Comment:
how do you know that the error that occurred is because the like was not found? It could be anything.

This is a little bit better: 

	try {
		const likes = await this.likeController.getLikes()

		return res.send(likes);

	} catch (e) {
              console.error(e.message)
		return res.status(500).send({msg: 'Something went wrong'})
	}

in this specific scenario, if there are no likes then you can just return an empty array so there is no need to cover for not found exceptions